docs/topics/proc_deploying-an-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,115 @@
+// parameter-runtime-name: Runtime specific atribute for substituting the approrpiate expanded rumtime name.
+// parameter-link-appdev-guide: link to the procedure providing a complete example of the appropriate runtime-specific POM file configuration.
+//  context: used in anchor IDs to conflicts due to duplicate IDs.
+//
+// Rationale: This procedure is the same for 2 or more runtimes.
+[id='deploying-an-application-to-standalone-rhel_{context}']
+= Deploying an application to standalone Red Hat Enterprise Linux
+
+Deploy your {parameter-runtime-name} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* A Maven-based {parameter-runtime-name} application.
+* The Red Hat Maven repository link:{parameter-link-appdev-guide}[specified] in the `pom.xml` of your project.
+* RHEL 7 installed.
+* OpenJDK 8 installed.
+* Maven 3.5 or later installed.
+
+
+.Procedure
+
+. Navigate to the directory with your application project:
++
+[source,bash,subs="attributes+",options="nowrap"]
+----
+$ cd myApp
+----
+
+. Ensure that you properly configure the {parameter-runtime-name} Maven plugin:
++
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<project ... >
+ifdef::built-for-spring-boot,built-for-spring-boot-1-5[]
+  ...
+  <!-- Specify target artifact type for the repackage goal. -->
+  <packaging>jar</packaging>
+endif::[]
+  ...
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>{parameter-group-name-runtime-plugin}</groupId>
+        <artifactId>{parameter-artifact-name-runtime-plugin}</artifactId>
+        <version>{parameter-version-runtime-plugin}</version>
+ifdef::built-for-vertx[]
+        <executions>
+          <execution>
+            <id>vmp</id>
+            <goals>
+              <goal>initialize</goal>
+              <goal>package</goal>
+            </goals>
+          </execution>
+        </executions>
+endif::[]
+ifdef::built-for-thorntail[]
+        <executions>
+          <execution>
+            <goals>
+              <goal>package</goal>
+            </goals>
+          </execution>
+        </executions>
+endif::[]
+ifdef::built-for-spring-boot,built-for-spring-boot-1-5[]
+        <executions>
+          <execution>
+              <goals>
+               <goal>repackage</goal>
+             </goals>
+          </execution>
+        </executions>
+endif::[]
+      </plugin>
+    </plugins>
+  </build>
+  ...
+</project>
+----
+
+. Package your application:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ mvn clean package
+----
++
+Consider packaging your application as a fat jar to include all of the necessary dependencies.
+ifdef::built-for-vertx[The Vert.x Maven Plugin packages applications as fat jars link:https://reactiverse.io/vertx-maven-plugin/#introduction[by default].]
+ifdef::built-for-thorntail[You can use the Thorntail Maven Plugin to package you application as an link:https://docs.thorntail.io/2.4.0.Final/#uberjar_thorntail[_uberjar_].]
```

Comment:
The link should point to the uberjar module in our runtime guide. I am not sure about the other runtimes, but those links may be OK.

Reply:
Thank you for the comment. Thee Spring Boot and Vert.x runtime plugins are not productized, and therefore the linked instructions have no counterpart in the Launcher or Product docs. I did, however update the Thorntail link so that now it points to Launcher docs.

Reply:
I've just checked with Ali Heslin, and she says that the links to the other runtimes' doc sets in this context are perfectly acceptable.